src/views/Citing.md

```diff
@@ -0,0 +1,132 @@
+<div
+  class="container-fluid monarch-view monarch-about-view">
+
+#### Goals
```

Comment:
Looks like you copied AboutMonarch.md, but didn't delete any of the original content. I think there is content and CSS here that is not needed for your How to Cite text.